### ISSUE TITLE
Selectpicker when virtualized set, items overlaps each other

### ISSUE THREAD

**Issue by thaynarbo** [rsuite/rsuite#3983]
### What version of rsuite are you using?

5.70.0

### What version of React are you using?

18.3.1

### What version of TypeScript are you using (if any)?

5.4.2

### What browser are you using?

Chrome

### Describe the Bug

When I set virtualized the items overlap one another. Is there a way I can add more space between them? tried the renderMenuItem but it did not have any effect. 

<img width="941" alt="image" src="https://github.com/user-attachments/assets/7558f8e0-ee53-49f5-a984-f7488772ca05">

### Expected Behavior

I should be able to set the gap between the items.

### To Reproduce

https://react-6fg4vb.stackblitz.io

**Comment by Hedz0r** [comment]
Had the same issue. I used the `listProps.itemSize` property to set the height of virtualized items.

**Comment by simonguo** [comment]
This should be a bug. By default, the options will wrap automatically based on the text length, but after using virtualized, the line height must be fixed, resulting in text overlap. You can use the following CSS for temporary fix, and we will fix this issue in the next version.

```css
.rs-picker-select-menu-item {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}
```

**Comment by simonguo** [comment]
Fixed in v5.72.0
